### ISSUE TITLE
Add option to disable Minecraft integration within installer

### ISSUE THREAD

**Issue by samerton** [NamelessMC/Nameless#1592]
Add option to disable Minecraft integration within installer to prevent default admin account having ingame associations.

**Comment by Aberdeener** [comment]
Will be started after #2606 is complete

**Comment by Aberdeener** [comment]
Out of scope for 2.0.0-pr13 I believe
